_posts/2018-09-12-september-office-hours.md

```diff
@@ -0,0 +1,72 @@
+---
+title: September Office Hours
+description: Multi-user OpenFaaS Cloud, Istio Service Mesh, and Stateless Microservices
+
+date: 2018-09-11
+image: /images/september-office-hours/team.jpg
+categories:
+  - office-hours
+  - community
+author_staff_member: burton
+dark_background: true
+---
+
+The OpenFaaS community holds regular office hours for anyone to join in as well as specialized discussions with project members, and core contributors. During these discussions, we talk about what's new in the project, where we're headed, and any other updates individual members may have.
+
+All of the office hours meetings are recorded and available for later viewing on YouTube.
```

Comment:
Link to the playlist here.

Reply:
I wasn't able to find an actual playlist of the meetings recordings. They're all uploaded to your YouTube account, but not stored in a specific playlist. 

Should they be moved to the OpenFaaS channel?